cooltools/cli/compute_expected.py

```diff
@@ -70,6 +79,14 @@
     default="weight",
     show_default=True,
 )
+@click.option(
```

Comment:
this is taken from `cooler.balance` , not sure if `blacklist` is the best keyword here